MachineBuilder/machines/MotorBike.cs

```diff
@@ -29,5 +29,10 @@ public static float CalculateSpeed(int rpm, int gear)
             float speed = speedConst * rpm * gear;
             return speed;
         }
+        public void TravelDistance(int distance)
+        {
+            Console.WriteLine("{0} this distance was traveled.", distance);
+        }
```

Comment:
Do the bike spend some fuel while travelling? Do the motor bike's odometer changes its value as well?